### ISSUE TITLE
Autoleveler Broken ?

### ISSUE THREAD

**Issue by KevTN5656** [Qendolin/pokecord-bot#33]
I try to run `PCBot.AutoLeveler.start( 678 )` in the console and it returns `Promise {<resolved>: undefined}` not doing anything. Oh if you ever switch to visual photo based matching here are all the pokemon photos https://mega.nz/folder/puARRA6B

**Comment by Qendolin** [comment]
You have to supply a list of Pokemon numbers (the numbers when you do `p!pokemon`). So `PCBot.AutoLeveler.start( [678] )` would be correct. To level multiple Pokemon you can do `PCBot.AutoLeveler.start( [678, 679, 680] )`

By the way, I am doing "visual photo based matching" (perceptual matching) and the folder that you shared with me is password protected. But as far as I know pokecord uses the images at https://bulbapedia.bulbagarden.net/

**Comment by Qendolin** [comment]
I would love to have a more user friendly bot, however I have other things to do.

**Comment by KevTN5656** [comment]
https://mega.nz/folder/puARRA6B#adnxP8GN7AeT7ovgP3o2rg

There is the new link your choice to decide if you wanna scrap the shit out of bulbpadeia. TBH pretty friendly I just dont know how to script JS so I setup the array like I would in PY.

**Comment by Qendolin** [comment]
Thanks for the link. I know it's not too nice to scrape bulbpadeia but all images get hashed using the dHash algorithm and all the computed hashes are stored in this repo (/src/data/hashes.js). So basically I only need to scrape bulbapedia once and I think that's acceptable. Before I made this bot I looked at what's out there but most repos are unorganized, don't include the hashing code or use pHash (and there was no pHash js lib that I could use).
